### DISCUSSION TITLE
How to be notified by mail when a new discussion is opened?

### DISCUSSION THREAD

**Discussion by SchneiderInfosystems** [community/community#3669] [Discussions]
How to be notified by mail when a new discussion is opened?

As the author of a repository, is it possible to be notified of new discussions by e-mail? I would have expected this to work the same way as for issue notifications.

**Comment by PranamBhat** [comment]
Hi @SchneiderInfosystems 

Hope you have subscribed to notifications :


![image](https://user-images.githubusercontent.com/32120033/113256220-810a4780-92e6-11eb-9b15-7b22fb79b6ed.png)



![image](https://user-images.githubusercontent.com/32120033/113256278-92ebea80-92e6-11eb-8145-0bf641a06dee.png)




Also, you can customize notification settings for your organizations.

![Uploading image.png…]()

**Reply by PranamBhat** [reply]
![image](https://user-images.githubusercontent.com/32120033/113256417-c9c20080-92e6-11eb-8609-90f877cfa530.png)

**Comment by SchneiderInfosystems** [comment]
That is exactly what I have configured in my settings. Isn't there a special setting for discussions? I ask because I get all other notifications except for the discussion forum. I also checked the spam folder - nothing found.

**Comment by SchneiderInfosystems** [comment]
I have seen now, that there is a notification area on all discussions. How can I control, that newly created discussions are per default set to "subscribed"?
![image](https://user-images.githubusercontent.com/9332047/113274132-a181d400-92dd-11eb-9d72-2cc22d60512c.png)

**Comment by SchneiderInfosystems** [comment]
For me, this point is still not solved yet. I miss sometimes new open discussions by other users in my own repository. That is bad.

**Reply by verdverm** [reply]
![image](https://user-images.githubusercontent.com/1390600/129611395-41e4d7bb-7440-4d29-a4fa-73229186b636.png)

Have you subscribed to "All Activity"? (your screen shot looks to indicate that this is not the case)

This is sufficient for me to receive notifications of discussions for repositories I am not a member of (and also my own repos). I receive both emails and have entries in the GH notification area

**Reply by SchneiderInfosystems** [reply]
Thank you for this valuable hint. No, my setting was to participating and @mentions. I was searching in the settings of my account and the repository for a such setting.
